src/frontend/state/GlobalState.tsx

```diff
@@ -295,7 +295,7 @@ export class GlobalState extends PureComponent<Props> {
   }
 
   handleSuccessfulLogin = (runner: Runner) => {
-    this.handleCategory(runner)
+    this.handleCategory('all')
```

Comment:
Setting the category on `all` instead of runner works better imo.

Reply:
agree, this would fix https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1949